### ISSUE TITLE
maintainers' next meeting

### ISSUE THREAD

**Issue by raphaeldussin** [pangeo-data/xESMF#6]
@rabernat @huard @stefraynaud @andersy005 

we're starting to get some PRs in the pipeline, when would be a good time to do the next meeting?

**Comment by andersy005** [comment]
Would it be useful to set up a doodle https://doodle.com/ poll?

**Comment by andersy005** [comment]
http://whenisgood.net/ is another alternative

**Comment by rabernat** [comment]
Btw, you can tag @pangeo-data/xesmf to hit all the maintainers! I'd be happy to participate in another meeting.

**Comment by raphaeldussin** [comment]
@pangeo-data/xesmf please let me know when you're free to e-meet

http://whenisgood.net/zqgdrpk

PRs https://github.com/pangeo-data/xESMF/pull/1 https://github.com/pangeo-data/xESMF/pull/3 https://github.com/pangeo-data/xESMF/pull/4 are all mature but the "masking update" PR will likely need a small update once the "weights in memory" PR is merged. The last one is just additional documentation.

**Comment by raphaeldussin** [comment]
up

**Comment by stefraynaud** [comment]
Back from holidays, I can attend to the meeting too.

**Comment by rabernat** [comment]
Thanks for organizing this Raphael! I consider my role mainly as a coordinator. My goal is to help it transition from a one-person project to a community project. But I don't have time to work on xESMF directly. Therefore, I hope my presence is not necessary at the meeting. (Unfortunately I'm unavailable on Wednesday.)

You should all feel free to add to the meeting notes: https://docs.google.com/document/d/10J2b6z8tJ3TUJ-w01owwZQHm9c192qlKdjSHrPRG_ag/edit

I recommend appointing someone to chair the meeting (set the agenda, call on people, and keep things on schedule.)

I have heard nothing from @JiaweiZhuang for months.

**Comment by andersy005** [comment]
> Based on the poll, let's shoot for:

> Date: September 2, 2020
> Time: 3:00:00 PM

@raphaeldussin, what is the time zone? I am trying to determine whether I'll be able to attend.

**Comment by andersy005** [comment]
> Based on the poll, let's shoot for:

> Date: September 2, 2020
> Time: 3:00:00 PM ET

I have a time conflict on Wednesday 3pm ET :(

**Comment by raphaeldussin** [comment]
I can push it to 4PM ET if @huard and @stefraynaud are ok.
I don't think anyone else if joining

**Comment by raphaeldussin** [comment]
Next meeting scheduled for Sept 30 3:30PM ET

**Comment by huard** [comment]
Next meeting scheduled for Nov 09, 15h30 EST.

**Comment by raphaeldussin** [comment]
I have a conflict monday, I had it scheduled for Wednesday 3:30EST.
would Wed work?

**Comment by stefraynaud** [comment]
Ok for me Wednesday 3:30EST.

**Comment by raphaeldussin** [comment]
@huard @stefraynaud should we touch base on the current issues, say early september?

**Comment by stefraynaud** [comment]
Hey guys. What is your availability for a meeting?

**Comment by huard** [comment]
Early next week ?

**Comment by raphaeldussin** [comment]
Monday 10AM would work for me

**Comment by huard** [comment]
In light of the issues with 0.6.1 and chunking the weight with the sparse backend, I suggest we roll back the changes made in 0.6.1 and release a 0.6.2 with the other PRs merged since then, in particular the bug fix for SpatialAverager. 
We'll plan a 0.7.0 release using the sparse backend. 

I've looked into [marking conda packages as broken](https://github.com/conda-forge/admin-requests), but I'm not sure it's worth the trouble here. I suggest we add a note in the CHANGES.txt highlighting the issues.

**Comment by raphaeldussin** [comment]
@huard @aulemahal are you guys free tomorrow for a quick meeting?

**Comment by aulemahal** [comment]
Yes! Any time between 10 and 18 (UTC-5). The later the more chance I will have had time to work on the suggestions of dcherian (that could fix xesmf with sparse instead of having to roll back).

**Comment by huard** [comment]
Yes, only busy at 12h (ET).

**Comment by huard** [comment]
I sent a request to ReadtheDocs to unlink https://xesmf.readthedocs.io/.

If you agree, I would also send a request to github to detach the pangeo-data fork from JiaweiZhuang/xESMF.

**Comment by raphaeldussin** [comment]
wouldn't it create a confusion to detach? right now people can see the pangeo fork is a continuation of Jiawei's original work

**Comment by raphaeldussin** [comment]
right!forgot about that. sure is annoying, go ahead then!

**Comment by huard** [comment]
Done.
